features/import.feature

```diff
@@ -343,6 +343,133 @@ Feature: cli-kintone import command
     Then I should get the exit code is non-zero
     And The output error message should match with the pattern: "Error: ENOENT: no such file or directory"
 
+  Scenario: CliKintoneTest-44 Should upsert the records successfully with --update-key is a Record_number field.
+    Given The app "app_for_upsert" has no records
+    And The app "app_for_upsert" has some records as below:
+      | Text   | Number |
+      | Alice  | 10     |
+      | Bob    | 20     |
+    And Load the record numbers of the app "app_for_upsert" as variable: "RECORD_NUMBERS"
+    And The csv file "CliKintoneTest-44.csv" with content as below:
+      | Record_number      | Text   | Number |
+      | $RECORD_NUMBERS[0] | Lisa   | 30     |
+      | $RECORD_NUMBERS[1] | Rose   | 40     |
+      |                    | Jenny  | 50     |
+    And Load app ID of the app "app_for_upsert" as env var: "APP_ID"
+    And Load app token of the app "app_for_upsert" with exact permissions "view,add,edit" as env var: "API_TOKEN"
+    When I run the command with args "record import --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --update-key Record_number --file-path CliKintoneTest-44.csv"
+    Then I should get the exit code is zero
+    And The app "app_for_upsert" should has 3 records
+    And The app "app_for_upsert" should has records as below:
```

Comment:
```suggestion
    And The app "app_for_upsert" should have records as below:
```
`should has` or `should have`
Ref: https://prowritingaid.com/have-vs-has#:~:text=Is%20It%20%22She%20Should%20Have,because%20it%27s%20a%20modal%20verb.